warwomb/data_files/warcaster_units.txt

```diff
@@ -0,0 +1,643 @@
+[
+    {
+        "factions": ["AC", "EMP", "MW", "ISA"],

```

Comment:
Could we spell these out? "Aeternus Continuum", "Empyrian", etc?

Reply:
See comment below.